x-pack/plugins/observability_solution/apm/public/components/shared/add_data_buttons/buttons.tsx

```diff
@@ -34,22 +34,24 @@ export const collectServiceLogs = {
   link: '/app/observabilityOnboarding/customLogs/?category=logs',
 };
 
-export function AddApmData({
-  onClick,
-  ...props
-}: {
+interface AddApmDataProps {
   onClick?: () => void;
   'data-test-subj': string;
-}) {
+  fill?: boolean;
```

Comment:
You can use EuiButtonProps from EUI to extend the props

Reply:
Thanks for the suggestion!
I'll be adding the `EuiButtonSize` to the `size` but I don't really want to extend all the `EuiButtonProps`.